src/Formatter/Formatter.php

```diff
@@ -225,7 +266,7 @@ public function onBeforeOutlineTested(BehatEvent\BeforeOutlineTested $event)
         $scenario->setName($event->getOutline()->getTitle());
         $scenario->setTags($event->getOutline()->getTags());
         $scenario->setLine($event->getOutline()->getLine());
-        $scenario->setType('scenario_outline');
+        $scenario->setType('scenario');
```

Comment:
I will also check this to be compatible with the Jenkins plugin I mentioned before.

Reply:
See above